src/Collector/ExcludeInternalCollector.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Collector;
+
+use Qossmic\Deptrac\AstRunner\AstMap;
+use Qossmic\Deptrac\AstRunner\AstMap\AstClassReference;
+
+class ExcludeInternalCollector implements CollectorInterface
```

Comment:
I will rename this to `MarkedInternalCollector` instead. For excluding them this can then be used inside bool's `must_not`.